index.js

```diff
@@ -0,0 +1,138 @@
+const menuIcon = document.querySelectorAll('#menu-icon');
+
+menuIcon.forEach((icon) => {
+  icon.addEventListener('click', () => {
+    document.querySelector('.mobile-nav').style.display = 'flex';
+    document.body.style.overflow = 'hidden';
+  });
+});
+
+const closeMenu = () => {
+  document.querySelector('.mobile-nav').style.display = 'none';
+  document.body.style.overflow = 'scroll';
+};
+
+closeMenu();
+
+// featured speakers section
+const featuredSpeakers = document.querySelector('.feature');
+featuredSpeakers.classList.add('featured-speakers');
+document.body.appendChild(featuredSpeakers);
+
+// // featured title
+// const featuredTitle = document.createElement('h2');
+// featuredTitle.classList.add('speakers-title');
+// featuredTitle.textContent = 'Featured Speakers';
+// featuredSpeakers.appendChild(featuredTitle);
```

Comment:
- Hey, please, let's remove these commented pieces of code to make your codebase clean and maintainable.

Reply:
@zieeco done with this change